run_kraken.py

```diff
@@ -12,6 +12,96 @@
 import pyfiglet
 
 
+# Function to publish kraken status to cerberus
+def publish_kraken_status(config, failed_post_scenarios):
+
+    if config["cerberus"]["cerberus_enabled"]:
+        cerberus_url = config["cerberus"]["cerberus_url"]
```

Comment:
Let’s make cerberus_integration a function as we will need to consume True/False signals after every chaos scenario (pod scenario, node scenario, cluster shutdown scenario, etc). I have added a [cerberus_integration](https://github.com/openshift-scale/kraken/pull/10/files#diff-cfec6e969439720ae75ffa7156ac298bR17) function in the node scenarios PR to achieve the same, it can be modified to return True/False signals additionally to fit in with the pod scenarios as well.

Reply:
It would be great if this could be done.

Reply:
ah yes ok sorry I missed this

Reply:
@yashashreesuresh we have pretty much the same function here. I'm a little unclear on what you wanted me to change here? Just the name of the function?

Reply:
A function to be defined which when called returns Cerberus true/false signal alone. In the current implementation of _publish_kraken_status_, retrieval of status from Cerberus is part of the entire code. I was telling about separating the retrieval of Cerberus status part.

Reply:
Because post actions works only with pod scenarios for now. In the above function, _failed_post_scenarios_ is included which is for pod scenarios and is not applicable for node scenarios and hence we need a separate function for obtaining true/false from cerberus.

Reply:
changed it up! what do you think?

Reply:
Looks good! Thanks for the change!